src/storybook/story-test-runner/extend-stories.ts

```diff
@@ -58,13 +79,20 @@ function getStoryNameId(storyName: string): string {
     return storyName.replace(nonAsciiWordRegExp, "").toLowerCase();
 }
 
-function getStoryFile(storyPath: string): StoryFile {
+function getStoryFile(storyPath: string): StoryFile | null {
     const unmockFn = mockLoaders();
-    const storyFile = require(storyPath); // eslint-disable-line @typescript-eslint/no-var-requires
+
+    let storyFile;
+
+    try {
+        storyFile = require(storyPath); // eslint-disable-line @typescript-eslint/no-var-requires
+    } catch (_) {} // eslint-disable-line no-empty
+
+    const isStoryFileFailedToLoad = !storyFile || typeof storyFile === "function";
```

Comment:
though, if it is, but we can't load it (`.tsx` case), we will get `mockedModuleProxy`, which is a function.